internal/pkg/agent/cmd/command_components_fips_test.go

```diff
@@ -0,0 +1,109 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+//go:build requirefips
```

Comment:
fips otel tests do not check for prometheus as it was disabled in https://github.com/elastic/elastic-agent/pull/7548